### PR TITLE
Updated tests to support parallel browser running

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates integration tests to support parallel browser running**

Also:
- removed page deletion test (needed for proper parallel running)

### CODE REVIEW

File: aldryn_events/tests/frontend/protractor.conf.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ var config = {
     // Maximum number of total browser sessions to run. Tests are queued in
     // sequence if number of browser sessions is limited by this parameter.
     // Use a number less than 1 to denote unlimited. Default is unlimited.
-    maxSessions: 1,
+    maxSessions: 10,
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/protractor.conf.js]: we can remove this at some point again

**Comment by FinalAngel** [comment]
LGTM at first sight @vxsx ?

**Comment by vxsx** [comment]
codewise looks good, but why is it successful on travis even though tests actually failing

**Comment by Marketionist** [comment]
Looking at the travis logs I'd say that there is a problem with running tests while not all migrations finished and server didn't got launched

**Pull request merged**